src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/ContentEncryptionAlgorithmTests.cs

```diff
@@ -60,6 +60,53 @@ private static void VerifyAlgorithmRc2_128(byte[] encodedMessage)
         }
 
         [Fact]
+        public static void DecodeAlgorithmRc2_40_FixedValue()
+        {
+            ContentInfo expectedContentInfo = new ContentInfo(new byte[] { 1, 2, 3, 4 });
+            byte[] encodedMessage =
+                ("3082011806092A864886F70D010703A0820109308201050201003181CC3081C90201003032301E311C301A0"
+                + "60355040313135253414B65795472616E73666572436170693102105D2FFFF863BABC9B4D3C80AB178A4CCA"
+                + "300D06092A864886F70D010101050004818004E46A48651034B01134B0D4F665C9E85F6C45B58458ECDBAFE"
+                + "B6B55CBFA9AEBEFA52BCBEF3C8811B5118970562623FC35D4B733B55CBC50DA4F49822E1D198834897D3540"
+                + "7B329FECF49277159F2FEAB31173004776B03746381E0DA660B6D656A861E54E79186F36F450105DEB2714D"
+                + "02DB5500921EBE4F1A7D3DFB07E4EE9303106092A864886F70D010701301A06082A864886F70D0302300E02"
+                + "0200A00408D621253C94AF659B800802930ACE6A997122").HexToByteArray();
+            EnvelopedCms ecms = new EnvelopedCms();
+            ecms.Decode(encodedMessage);
+
+            AlgorithmIdentifier algorithm = ecms.ContentEncryptionAlgorithm;
+            Assert.NotNull(algorithm.Oid);
+            Assert.Equal(Oids.Rc2, algorithm.Oid.Value);
+            Assert.Equal(40, algorithm.KeyLength);
+        }
+
+        [Fact]
+        [ActiveIssue(10310, PlatformID.AnyUnix)]
```

Comment:
This seems like a ConditionalFact instead of an ActiveIssue (and we should be testing the PNSE)